nixos/modules/programs/nist-feed.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed notifies you about the newest published CVEs.");
+      package = mkPackageOption pkgs "nist-feed" { };
+      arguments = mkOption {
+        type = types.str;
+        default = "-l -s CRITICAL";
+        description = mdDoc ''lib.
+          Arguments to provide to NIST-Feed.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Does this package provide some kind of cli interface? If it only gives notifications via libnotify, I think it would be better not to clutter the users environment.

Reply:
Yup, it can be used as stand-alone cli program, but also has an option for notificiations